wallet/how-to/use-sdk/gaming/unity/index.md

```diff
@@ -20,7 +20,7 @@ The MetaMask Unity SDK supports macOS, Windows, Linux, iOS, Android, and WebGL.
 The following video demonstrates how to install and use the MetaMask Unity SDK.
 
 <p align="center">
-  <ReactPlayer url='https://www.youtube.com/embed/0D1cIH-PZtI' />
+  <YoutubeEmbed url='https://www.youtube.com/embed/0D1cIH-PZtI' />
```

Comment:
This does not appear in the rendered preview and requires an import statement at the top of the page:

```
import YoutubeEmbed from '@site/src/components/YoutubeEmbed';
```